content/efcore/getting-started/samples/loading-related-data-tutorial/multiple-include-methods/Program.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+public class Program
+{
+    public static void Main()
+    {
+        using (var context = new LibraryContext())
+        {
+            var book = context.Books
+                .Single(b => b.BookId == 1)
+                .Include(b => b.Editions)
+                .Include(b => b.Author);
+            Console.WriteLine(String.Format("{0} - {1}", book.Author, book.Edition));
```

Comment:
If we want to show the actual data, we will need to change it so that it prints `book.Author.FirstName` and `book.Author.LastName`. We also should do a foreach loop or something of the sort for editions, since it is a list.